handlers/handler-match/src/main/java/org/apache/servicecomb/match/service/PolicyService.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.servicecomb.match.service;
+
+import org.apache.servicecomb.match.policy.Policy;
+
+public interface PolicyService {
+
+  Policy getCustomPolicy(String kind, String mark);
```

Comment:
<T extends Policy> getCustomPolicy(xxxx)
is better.

Reply:
seems like the same

Reply:
the comments do not show correctly. using `<T extends Policy> ` can avoid implicite casting.